repository/w.json

```diff
@@ -468,6 +468,17 @@
 				}
 			]
 		},
+		{
+			"name": "Wikidot Syntax highlighter",
```

Comment:
Language support packages should just be named after their language. They do a lot more than simply add colors. In this case simply "Wikidot".